IbisLib/application.cpp

```diff
@@ -240,6 +239,7 @@ Application::~Application()
     for( int i = 0; i < allPlugins.size(); ++i )
     {
         allPlugins[i]->BaseSaveSettings( settings );
+        allPlugins[i]->Delete();
```

Comment:
Are you sure we should delete plugins? My understanding is that since the plugins are instantiated by Qt, Qt is also probably taking care of deleting them. We might want to clear data that is referenced by plugins, but not delete them, no?